### DISCUSSION TITLE
Any way to get a panning effect using current effects available?

### DISCUSSION THREAD

**Discussion by chrsrns** [wwmm/easyeffects#1200] [Q&A]
I heard somewhere that the "direction" of where a music is coming from affects the focus of a person. I think this applies to me, as listening to headphone music distracts me, while speaker music doesn't

Using the available plugins, is there a way to pan the audio so that the headphone audio sounds like it's coming from the front of my head, instead of sounding like the sound is coming from inside of my head?

**Comment by wwmm** [comment]
I am not sure if it is possible to achieve exactly what you want with the plugins we currently have. Depending on the directional effects being targeted a HRTF plugin would be needed. But usually when using headphones for long periods of time the `crossfeed` plugin is recommended. It will combine a fraction of the left and right channel signal in a way that the result will be similar to listening to speakers. It is not a directionality control. What a crossfeed tries to fix is the unrealistic separation between the left and right channels that exists in a headphone. When listening to speakers our ears get a little of both channels and not the sound of only one of them.

Another filter to try is the `Stereo Tools`. Try to play a little with the balance parameter. Depending on what you are listening to you can make the voice be a little more to the let or to the right. Playing with the parameters in the `Stereo Matrix` tab may help too.

**Reply by wwmm** [reply]
And in case someone out there have made a stereo impulse response file that achieves the kind of thing you want you can try to load it in our convolver. So searching for impulse response files that aim to make the headphone experience more pleasant may help too.

**Reply by chrsrns** [reply]
> And in case someone out there have made a stereo impulse response file that achieves the kind of thing you want you can try to load it in our convolver. So searching for impulse response files that aim to make the headphone experience more pleasant may help too.

Alright. I'll try to search for something like that.
